utilities/transactions/transaction_lock_mgr.h

```diff
@@ -27,7 +27,7 @@ struct LockMap;
 struct LockMapStripe;
 
 class Slice;
-class TransactionDBImpl;
+class PessimisticTxnDB;
```

Comment:
I guess the decision to use `PessimisticTxn` was already taken in a different PR, but I would suggest to be consistent with the existing name which is `OptimisticTransaction`

so I would suggest using `PessimisticTransaction` and `PessimisticTransactionDB`